src/components/Avatar/Avatar.tsx

```diff
@@ -3,9 +3,18 @@ import { AvatarProps } from './types';
 import React from 'react';
 import guy from './images/guy';
 
-const Avatar: React.FC<AvatarProps> = ({ theme, image, text }: AvatarProps) => {
+const Avatar: React.FC<AvatarProps> = ({
+    theme,
```

Comment:
please make the props in alphabetical order. there is VSCode plugin called `sortLines`